assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -65,30 +75,50 @@ const baseInitialState = {
 
 const baseActions = {
 	/**
-	 * Sets user input settings.
+	 * Gets cached user input settings and save them to the data store.
 	 *
-	 * @since 1.19.0
+	 * @since n.e.x.t
```

Comment:
Since this action is documented, it should be marked as private, since it's a low-level action that should only be used via the `getUserInputSettings` resolver. The alternative is to not document it at all.